### PR TITLE
style: refactor SCSS queries

### PR DESCRIPTION
**Pull request by marissahuysentruyt**
### Description:

Refactoring was done in the SCSS files to ensure that media queries were not far removed from the elements they target. I divided up several long queries into shorter ones, and nested them within the selectors. 

### Spec:

Designs: [Figma file](https://www.figma.com/file/UjMQEwUvAsnFAMhYNdUiVT/Apprenticeship-Capstone?node-id=84%3A270&t=Z1mZXyHn90lItsyX-0)

See Story: [FSA22V2-297](https://sparkbox.atlassian.net/browse/FSA22V2-297)

### Validation:

- [x] This PR has code changes, and our linters still pass.

#### To Validate:

1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, etc).
2. Pull down all related branches.
3. Run `npm install` to make sure you have all proper dependencies. (this project requires Node 16+)
4. Copy the `.env.example` file, making sure it's in the root of your project, and rename it `.env.local`. Search for "Accessible Components" in 1Password to find the API Key and Base ID. Add both of those secrets to your newly created `.env.local` file. That should establish your connection to our Airtable database.
5. Run `npm run lint` to confirm no errors.
6. Run `npm run test` to confirm all tests pass.
7. In terminal: `npm run dev`.
8. Check [localhost:3000](http://localhost:3000/) to see changes.
9. Run axe Devtools on affected pages to confirm no urgent errors.
10. Spot check homepage and interior pages to make sure nothing has visually broken. 

---

### CODE REVIEW

File: src/styles/components/_usage-guidelines.scss [hunk 1]
```diff
@@ -14,24 +14,50 @@ $breakpoint-usage-guidelines-display: 70rem;
     color: var(--font-color-dark);
   }
 
+  @media (min-width: $breakpoint-usage-guidelines-display) {
```
Comment (mkendall-goodyear) [inline src/styles/components/_usage-guidelines.scss:17]: Nice refactor on this file!

File: src/styles/globals.scss [hunk 2]
```diff
@@ -18,13 +18,13 @@
...
+@import "./components/component-grid";
 
 /* COMPONENTS: most of the work goes here! BEM classes, headers/footers/carousels, etc */
 @import "./components/home";
 @import "./components/details-page";
 @import "./components/header";
 @import "./components/component-card";
-@import "./components/component-grid";
 @import "./components/footer";
 @import "./components/layout";
```
Comment (mkendall-goodyear) [inline src/styles/globals.scss:29]: Should the layout component also move to the Layout/Objects section?

Comment (MLKendall) [inline src/styles/globals.scss:29]: Should the layout file also be in the Layout section?

**Comment by MLKendall** [comment]
This looks good, Marissa!  Nice refactor.  I just left one question.

File: src/styles/globals.scss (hunk 2 above)
Comment (marissahuysentruyt) [reply src/styles/globals.scss:29]: Good call on that. I'll change it.

**Review by MLKendall — DISMISSED** [review]
Looks good, Marissa!

**Review by jkohrt7 — APPROVED** [review]
This looks great, especially combined with the new breakpoint variables. Good work!

**Review by jkohrt7 — APPROVED** [review]

**Pull request merged**